src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int FIGURES_NUMBER = 6;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[FIGURES_NUMBER];
+
+        for (int i = 0; i < figures.length; i++) {
+            if (i < figures.length / 2) {
+                figures[i] = figureSupplier.getRundomFigure();
```

Comment:
There is a typo in the method call `getRundomFigure()`. It should be corrected to `getRandomFigure()` to match the method name in the `FigureSupplier` class.